go-controller/pkg/ovn/obj_retry.go

```diff
@@ -107,8 +113,11 @@ func (r *retryObjs) initRetryObjWithAdd(obj interface{}, key string) {
 	}
 }
 
-// initRetryWithDelete tracks an object that failed to be deleted to potentially retry later
-// initially it is marked as skipped for retry loop (ignore = true)
+// initRetryWithDelete creates a retry entry for an object that is being deleted,
+// so that, if it fails, the delete can be potentially retried later.
+// initially it is marked as skipped for the retry loop (ignore = true).
+// When applied to pods, we include the config object as well in case the namespace is removed
```

Comment:
When applied to network policy?

Reply:
Let me add that part as well for completeness.